updating/updating-restricted-network-cluster/restricted-network-update-osus.adoc

```diff
@@ -6,7 +6,21 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-To get an update experience similar to connected clusters, you can use the following procedures to install and configure the OpenShift Update Service in a disconnected environment.
+To get an update experience similar to connected clusters, you can use the following procedures to install and configure the OpenShift Update Service (OSUS) in a disconnected environment.
+
+The following steps outline the high-level workflow on how to update a cluster in a disconnected environment using OSUS:
+
+. Configure access to a secured registry.
+
+. Update the global cluster pull secret to access your mirror registry.
+
+. Install the OSUS operator.
+
+. Create a graph data container image for OSUS.
+
+. Install the OSUS application and configure it to use your graph data image and your mirror registry.
```

Comment:
I would prefer "Install the OSUS application and configure your clusters to use the local OSUS"

Reply:
Sounds good, I implemented this change